src/content/smart-bulletins/031.md

```diff
@@ -0,0 +1,103 @@
+Due to the issuance of Executive Order (EO) 13834, “Efficient Federal Operations,” dated May 17, 2018, and the revocation of EO 13693, “Planning for Federal Sustainability in the Next Decade,” dated March 19, 2015, Section 10.2 is deleted in its entirety are replaced with the following: 
+
+EO 13834 affirms that it is the policy of the United States that agencies meet energy and environmental performance statutory requirements in a manner that increases efficiency, optimizes performance, eliminates unnecessary use of resources, and protects the environment. In implementing this policy, agencies are tasked to prioritize actions that reduce waste, cut costs, enhance the resilience of Federal infrastructure and operations, and enable more effective accomplishment of its mission. EO 13834 may be viewed at: https://www.fedcenter.gov/programs/eo13834.
+
+To assist agencies in meeting statutory and optional sustainable purchasing requirements, GSA offers an on-line Green Procurement Compilation. The Green Procurement Compilation is a comprehensive green purchasing resource designed for federal contracting personnel and program managers. It consolidates and organizes information from federal environmental programs in one place, saving you from visiting multiple websites. 
+
+The Green Procurement Compilation allows users to:
+- Quickly identify federal green purchasing requirements for the products and services;
+- Search by keyword or browse by category to find products and services;
+- Determine procurement options available to federal buyers, including applicable GSA Multiple Award Schedules (MAS), Federal Strategic Sourcing Initiative (FSSI) solutions, and GSA Global Supply;
+- Learn more about federal environmental programs, such as WaterSense and Bio-Preferred; and
+- Discover optional environmental programs and additional procurement guidance that will help meet sustainability goals. 
+
+The green products portal identifies all product types covered by mandatory and non-mandatory federal environmental programs, including: 
+- Bio-Preferred;
+- Comprehensive Procurement Guidelines;
+- Safer Choice;
+- Energy Star;
+- EPEAT;
+- Federal Energy Management Program (FEMP);
+- Significant New Alternatives Policy (SNAP); and
+- WaterSense.
+
+In order to determine which requirements apply to the products planned for purchase, the portal allows users to search or browse for the product.
+
+The services portal contains green purchasing guidance for services, including:
+- Cafeteria & Food Services;
+- Electronic Leasing;
+- Fleet Maintenance;
+- Janitorial Services;
+- Landscaping Services;
+- Laundry Services;
+- Meeting & Conference Services;
+- Pest Management; and
+- Transportation Services.
+
+### Attachments A.1 Glossary
+### Attachment A.6 – Disciplinary Category Definitions Guide
+For consistency, the following existing definitions contained in both of these attachments are deleted and replaced with the following: 
```

Comment:
For consistency, the following existing definitions contained in both of these attachments are deleted and replaced with the following:
A.1 External Fraud - Any felonious act of corruption or attempt to cheat the Government or corrupt the Government’s agents by someone other than Government officials. Otherwise known as third-party fraud.
A.1 Misuse - Unintentional use of the purchase, travel, fleet, and/or integrated charge card in violation of the FAR, DFARs, FTR, Agency Supplements, or Agency Policy/Procedures. These actions are the result of ignorance and/or carelessness, lacking intent, to include honest mistakes..